src/DotvvmAcademy.Web/Course/010_concepts/en/50_commands.md

```diff
@@ -10,19 +10,20 @@ CodeTask:
 
 # Commands
 
-__Commands__ are a kind of bindings used to invoke ViewModel's methods.
+The __Commands binding__ allows to call the ViewModel methods from the View.
 
-Unlike Value bindings, they make a request to the server and make necessary updates afterwards. Commands can be used in a control like the `Button`:
+Unlike the value bindings, the command bindings make a request to the server and perform the necessary updates afterwards. The commands can be used from many controls, the `Button` for example:
```

Comment:
Too many "the"s. "Unlike value bindings, command bindings..." is just fine. "Commands can be used from many controls like the `Button` for example:"